client/app/scripts/superdesk/editor/editor.js

```diff
@@ -285,7 +285,9 @@ function EditorService(spellcheck, $rootScope, $timeout, $q) {
         var index, text;
         while (tree.nextNode()) {
             text = tree.currentNode.textContent;
-            while ((index = text.indexOf(needle)) > -1) {
+            while ((index = self.settings.findreplace.caseSensitive ?
```

Comment:
because of performance it might be better to do it before the loop, so
```
var needle = caseSensitive ? needle : needle.toLowerCase();
...
while (tree.nextNode()) {
    text = caseSensitive ? tree.currentNode.textContent : tree.currentNode.textContent.toLowerCase();
    while ((index = text.indexOf(needle)) > -1) {
      ...
```
but didn't profile it (yet) so maybe it's not worth it